Menu/menu.asm

```diff
@@ -14,13 +14,14 @@ pushpc
 ; update in game hud colors 
 org $1BD662 : dw hexto555($814f16), hexto555($552903)
 org $1BD66A : dw hexto555($d51d00), hexto555($f9f9f9)
-org $1DB672 : dw hexto555($d1a452), hexto555($f9f9f9)
+org $1DB672 : dw hexto555($d0a050), hexto555($f9f9f9)
 org $1DB67A : dw hexto555($5987e0), hexto555($f9f9f9)
 org $1DB682 : dw hexto555($7b7b83), hexto555($bbbbbb)
 org $1DB68A : dw hexto555($a58100), hexto555($dfb93f)
 
 ; Free ROM in Bank 00
-org $0098AB : db $D8>>1
+org $0098AB : db $6C

```

Comment:
What are these for?

Reply:
These part of a weird table the game uses to control tile screen upload, it comes from this:
```
; $001888-$0018AA DATA
{
    db $00, $00, $04, $08, $0C, $08, $0C, $00
    db $04, $00, $08, $04, $0C, $04, $0C, $00
    db $08, $10, $14, $18, $1C, $18, $1C, $10
    db $14, $10, $18, $14, $1C, $14, $1C, $10
    db $18, $60, $68
}

; $0018AB-$0018BF - NULL
{
    db $FF, $FF, $FF, $FF, $FF, $FF, $FF, $FF
    db $FF, $FF, $FF, $FF, $FF, $FF, $FF, $FF
    db $FF, $FF, $FF, $FF, $FF
}
```

when you store a value into $0116 it is used as an index to then store into the actual register here:
```
    ; $1888, X that is
    ; Sets the high byte of the Target VRAM address.
    LDX $0116 : LDA.w $9888, X : STA $2117
```
instead of just storing the value you actually need directly from $0116 like a normal person 🙄